### PR TITLE
* Fix compilation warning due to SignalRSample.Web not being signed

### PR DESCRIPTION
**Pull request by pranavkm**
* Changes to run unit tests on desktop + coreclr

### CODE REVIEW

**Comment by pranavkm** [comment]
cc @moozzyk

File: test/Microsoft.AspNetCore.SignalR.Server.Tests/TaskAsyncHelperFacts.cs [hunk 1]
```diff
@@ -1,3 +1,4 @@
+#if NET451
```
Comment (pranavkm) [inline test/Microsoft.AspNetCore.SignalR.Server.Tests/TaskAsyncHelperFacts.cs]: Couldn't get these tests to pass on dotnet.

File: test/Microsoft.AspNetCore.SignalR.Server.Tests/PersistentConnectionFacts.cs [hunk 2]
```diff
@@ -67,12 +68,7 @@ public void UncleanDisconnectFiresOnDisconnected()
                 protectedData.Setup(m => m.Unprotect(It.IsAny<string>(), It.IsAny<string>()))
                              .Returns<string, string>((value, purpose) =>  value);
 
-                var connection = new Mock<PersistentConnection>() { CallBase = true };
-                var onDisconnectedCalled = false;
-                connection.Protected().Setup("OnDisconnected", context.MockRequest.Object, "1", false).Callback(() =>
```
Comment (pranavkm) [inline test/Microsoft.AspNetCore.SignalR.Server.Tests/PersistentConnectionFacts.cs]: Might be a bug in `Moq.Protected` in CoreCLR - it couldn't discover the method.

**Comment by BrennanConroy** [comment]
We don't care about running these tests, in fact we would prefer not running these tests

**Comment by pranavkm** [comment]
Can we delete it in that case?

**Comment by BrennanConroy** [comment]
Delete what?
@DamianEdwards @davidfowl

**Comment by pranavkm** [comment]
The tests? If we don't need to run them, why not delete them? They'll be in git history to revive if we ever need to

File: test/Microsoft.AspNetCore.SignalR.Server.Tests/TaskAsyncHelperFacts.cs (hunk 1 above)
Comment (moozzyk) [inline test/Microsoft.AspNetCore.SignalR.Server.Tests/TaskAsyncHelperFacts.cs]: What was the error? File a bug and add a comment to the code.

Comment (moozzyk) [inline test/Microsoft.AspNetCore.SignalR.Server.Tests/TaskAsyncHelperFacts.cs]: ..  or just remove the #ifdef - I think we are not running these tests anywhere

**Comment by moozzyk** [comment]
I'd rather not delete these tests.

File: test/Microsoft.AspNetCore.SignalR.Server.Tests/TaskAsyncHelperFacts.cs (hunk 1 above)
Comment (pranavkm) [inline test/Microsoft.AspNetCore.SignalR.Server.Tests/TaskAsyncHelperFacts.cs]: > I think we are not running these tests anywhere

I made them work though. Why do we have tests that don't run?

Comment (moozzyk) [inline test/Microsoft.AspNetCore.SignalR.Server.Tests/TaskAsyncHelperFacts.cs]: They were failing and fixing them was not a priority due to SignalR not being part of the first release.

Comment (moozzyk) [inline test/Microsoft.AspNetCore.SignalR.Server.Tests/TaskAsyncHelperFacts.cs]: Yes - once we get back to SignalR server we want these tests. This is why I don't want them to be deleted. On the other hand #ifdef  without any bug/comment could give an impression that the tests were not meant to run on NET451 which is not the case.

**Comment by pranavkm** [comment]
Updated.

**Comment by moozzyk** [comment]
:ship::it:

**Pull request merged**